pyramid_heroku/migrate.py

```diff
@@ -157,19 +157,20 @@ def migrate(self):
 
 def main():
     parser = argparse.ArgumentParser(
-        usage='python -m pyramid_heroku.migrate my_app etc/production.ini app:main',
+        usage='python -m pyramid_heroku.migrate my_app --ini_file=etc/production.ini --app_section=app:main',
```

Comment:
@zupo keyword arguments with default values is a native way to implement optional parameters. Consider a case when a user wants to specify only one optional value:

    # using defaults for both `ini_file` and `app_section`
    $ python -m pyramid_heroku.migrate my_app 

    # using default value for `app_section`
    $ python -m pyramid_heroku.migrate my_app --ini_file=etc/production.ini

    # using default value for `ini_file`
    $ python -m pyramid_heroku.migrate my_app --app_section=app:main

It's possible to achieve similar behavior with using positional arguments, but it will make CLI more confusing:
    
    # using defaults for both `ini_file` and `app_section`
    $ python -m pyramid_heroku.migrate my_app 

    # using default value for `app_section`
    $ python -m pyramid_heroku.migrate my_app etc/production.ini

    # using defaul for `ini_file` when user wants to provide value for `app_section` is not possible because of positional nature of arguments
    $ python -m pyramid_heroku.migrate my_app  etc/production.ini  app:main